### PR TITLE
EFRS-955 Fix organization is reload.

### PR DESCRIPTION
**Pull request by pbohdan**

### CODE REVIEW

File: ui/src/app/store/user/effects.ts [hunk 1]
```diff
@@ -87,7 +86,7 @@ export class UserListEffect {
             this.authService.logOut();
             return [];
           }
-          return [deleteUserSuccess({ userId }), loadApplications(), loadUsersEntityAction()];
```
Comment (isobolev) [inline ui/src/app/store/user/effects.ts:90]: as I remember loadApplications is fired to get correct roles. please check roles works correctly if you delete owner of organization; admin of org; user of org; owner of the app, etc. You can view in git history for which bug this line was added

Comment (pbohdan) [reply ui/src/app/store/user/effects.ts:90]: Fixed

Comment (isobolev) [reply ui/src/app/store/user/effects.ts:90]: what is the difference between `loadApplications` and `loadApplicationsNotLoader`?

Comment (pbohdan) [reply ui/src/app/store/user/effects.ts:90]: They differ from each other in that loadApplications() runs the preloader and loadApplicationsNotLoader() doesn't.

Comment (isobolev) [reply ui/src/app/store/user/effects.ts:90]: are you talking about setting `isPending: true`, in reducer by loadApplications? is this the only difference?

Comment (pbohdan) [reply ui/src/app/store/user/effects.ts:90]: Yes.

Comment (isobolev) [reply ui/src/app/store/user/effects.ts:90]: sounds like a 'plaster fix'. lets zoom and discuss

Comment (pbohdan) [reply ui/src/app/store/user/effects.ts:90]: Ok.

File: ui/src/app/store/application/effects.ts [hunk 2]
```diff
@@ -49,8 +50,8 @@ export class ApplicationListEffect {
 
   @Effect()
   loadApplications$ = this.actions.pipe(
-    ofType(loadApplications),
-    switchMap(action =>
+    ofType(loadApplications, loadApplicationsNotLoader),
```
Comment (isobolev) [inline ui/src/app/store/application/effects.ts:53]: what is the fix? loadApplicationsNotLoader and loadApplications are now identical

File: ui/src/app/store/app-user/effects.ts [hunk 3]
```diff
@@ -70,7 +70,6 @@ export class AppUserEffects {
     switchMap(action =>
       this.appUserService.deleteUser(action.applicationId, action.userId).pipe(
         map(() => deleteUserFromApplicationSuccess({ id: action.userId })),
-        tap(() => this.router.navigateByUrl(Routes.Home)),
```
Comment (pbohdan) [inline ui/src/app/store/app-user/effects.ts:73]: After deleting everything works as it should

**Review by endypanda — APPROVED** [review]

**Review by isobolev — APPROVED** [review]

**Pull request merged**